config/ulib_config.h

```diff
@@ -10,29 +11,29 @@ extern "C" {
 
 /**
  * @file ulib_config.h
- * 
+ *
  * @brief Configuration options for ulib
  */
 
 /*
- *  Define the heap 
+ *  Define the heap
  */
 
 /**
  * @brief   ulib malloc
  *
- *  Defines the malloc function that the ulib shall use as its own way to dynamically allocate
- *      memory from the HEAP. For simplicity, it can be defined as the malloc(size) from the `stdlib.h`.
+ * Defines the malloc function that the ulib shall use as its own way to dynamically allocate
+ * memory from the HEAP. For simplicity, it can be defined as the malloc(size) from the `stdlib.h`.
  */
-#define AZ_ULIB_CONFIG_MALLOC(size)    malloc(size)
+#define AZ_ULIB_CONFIG_MALLOC(size) malloc(size)
 
 /**
  * @brief   ulib free
  *
- *  Defines the free function that the ulib shall use as its own way to release memory dynamic 
- *      allocated in the HEAP. For simplicity, it can be defined as the free(ptr) from the `stdlib.h`.
+ * Defines the free function that the ulib shall use as its own way to release memory dynamic
+ * allocated in the HEAP. For simplicity, it can be defined as the free(ptr) from the `stdlib.h`.
  */
-#define AZ_ULIB_CONFIG_FREE(ptr)       free(ptr)
+#define AZ_ULIB_CONFIG_FREE(ptr) free(ptr)
```

Comment:
i'm curious if these should be outright removed since we don't use any malloc anywhere....

Reply:
It is not part of the PR, but I will remove it anyway to avoid further temptations. :)

---
In reply to: [338694623](https://github.com/Azure/azure-ulib-c/pull/40#discussion_r338694623) [](ancestors = 338694623)